app/javascript/react/screens/App/Overview/screens/MappingWizard/MappingWizardBody.js

```diff
@@ -35,32 +35,27 @@ class MappingWizardBody extends React.Component {
           {
             title: __('General'),
             render: () => <MappingWizardGeneralStep />,
-            onClick: () => console.log('on step 1 click'),
-            disabled: !this.props.mappingWizardGeneralStep.values
+            disableGoto: !this.props.mappingWizardGeneralStep.values
```

Comment:
i have no issue w/ this... mainly b/c `mappingWizardGeneralStep.values` is being inserted via parent props (and Redux) and localized to Mapping Wizard. There are no callbacks here, just redux-keyed props coming down the state tree. 😸

Reply:
`!this.props.mappingWizardGeneralStep.values` condition won't be enough to block a GoTo

`mappingWizardGeneralStep.syncErrors` is what decides if the current form is valid or not

Reply:
Right, but we want to allow the user to hop to/from forms that are invalid/empty/incomplete, as long as they have been visited already (and values gets initialized on first visit). We just don't want users getting to a step for the first time via any other means than the Next button (which does use that validation).